### ISSUE TITLE
link drag issue

### ISSUE THREAD

**Issue by pbuzdin** [ilyashubin/scrollbooster#27]
Hi, Ilya!

If there are links in draggable container, they will be fire click event while dragging.

**Comment by ilyashubin** [comment]
Hello! Right now click event is not prevented by default. Maybe its a good idea to prevent any clicks by default, I'll think about it, thanks
Right now you can do this:
```js
new ScrollBooster({
  ...
  onClick: (state, event) => {
    // prevent default link event
    const isLink = event.target.nodeName.toLowerCase() === 'link';
    if (isLink) {
      event.preventDefault();
    }
  }
});
```

**Comment by jos-** [comment]
It would be better to still allow normal clicks. See https://github.com/donmbelembe/vue-dragscroll/issues/61#issuecomment-583951401 for a possible solution.

**Comment by ilyashubin** [comment]
Good workaround, thank you! For now I implemented drag offset based solution instead of timer based, because I have access to those metrics
So `preventDefault` is called only when scrolled more than 5 pixels
https://github.com/ilyashubin/scrollbooster/commit/15f826449875f8fcfdcc558882a661971f68e96d

**Comment by jos-** [comment]
Nice, thanks for the fix.

**Comment by ilyashubin** [comment]
This fix is in v2.2.2, closing
